libs/components/modals/src/lib/modules/modal/modal.component.html

```diff
@@ -83,7 +83,9 @@
       [skyScrollShadowEnabled]="scrollShadowEnabled"
       (skyScrollShadow)="scrollShadowChange($event)"
     >
-      <ng-content select="sky-modal-content" />
+      <div skyLayoutHost [layout]="layout()">
```

Comment:
Can we eliminate the need for this extra `div` by moving `skyLayoutHost` up to the parent `div`?

Reply:
Good call. Removed the extra div.